Applications/Utils/FileConverter/MeshToRaster.cpp

```diff
@@ -0,0 +1,189 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#include <memory>
+#include <string>
+
+#include <tclap/CmdLine.h>
+
+#include <Applications/ApplicationsLib/LogogSetup.h>
+
+#include "BaseLib/BuildInfo.h"
+#include "GeoLib/AABB.h"
+#include "GeoLib/AnalyticalGeometry.h"
+#include "MeshLib/IO/readMeshFromFile.h"
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Node.h"
+#include "MeshLib/Elements/Tri.h"
+#include "MeshLib/Elements/Quad.h"
+#include "MeshLib/MeshSearch/MeshElementGrid.h"
+
+/// Returns the index of the element the given node is located in when projected onto a mesh.
+std::size_t getProjectedElement(std::vector<const MeshLib::Element*> const& elems,
```

Comment:
The function name doesn't express exactly what the comment says and what is returned.

Reply:
:heavy_check_mark: